lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -981,4 +981,11 @@ class SwiftASTContextForExpressions : public SwiftASTContext {
 };
 
 } // namespace lldb_private
+
+#ifndef NDEBUG
+/// Provided only for unit tests.
+template <typename SmallString>
+void ApplyWorkingDir(SmallString &clang_argument, llvm::StringRef cur_working_dir);
```

Comment:
From the looks of the function, it doesn't need to be a template, looks like `clang_argument` can be a `StringRef`.